lru.d.ts

```diff
@@ -1,21 +1,21 @@
-interface Entry<K,V> {
-  older :Entry<K,V>;
-  newer :Entry<K,V>;
+interface LRUCacheEntry<K,V> {
+  older :LRUCacheEntry<K,V>;
+  newer :LRUCacheEntry<K,V>;
   key   :K;
   value :V;
 }
 
-export class LRUCache<K,V> {
-  constructor(limit :number);
+interface LRUCacheStatic<K,V> {
```

Comment:
Why is this renamed and why is it an interface instead of a class? Can you please link to the appropriate documentation suggesting to use interfaces instead of classes in declaration files? Thanks.